README.md

```diff
@@ -1 +1,191 @@
-# pegnet-compose
+# PegNet Mining Tutorial
+
+PegNet Compose leverages Docker Compose to help you start and manage a PegNet miner.
+
+## Requirements
+
+-   Linux (Debian / Ubuntu preferred)
+-   Windows
+
+## Setup
+
+This is the initial setup process. You will only need to do this once.
+
+#### 1. Clone and cd into this repo
+
+```shell script
+git clone https://github.com/pegnet/pegnet-compose.git && cd pegnet-compose
+```
+
+#### 2. Install Docker and Docker-Compose (skip this if already installed)
+
+##### Linux:
+- run `sudo ./get_started.sh` (easiest way)
+
+or
+
+- copy and run the following commands:
+```shell script
+sudo apt-get update && sudo apt-get install curl python3 apt-transport-https ca-certificates software-properties-common -y && curl -fsSL get.docker.com -o get-docker.sh && sh get-docker.sh && sudo usermod -aG docker pi && sudo pip3 install docker-compose
+```
+
+
+After installing Docker it's better to reboot the machine
+
+##### Windows:
+- Install Docker and Docker Compose (https://docs.docker.com/docker-for-windows/install/)
+
+
+#### 3. Get the bootstrap file (skip this if you want to sync the blockchain from scratch)
+
+##### Linux:
+- run `cd volumes/factomd; sudo ./bootstrap.sh`
+
+##### Windows:
+- download [this file](https://factom-public-files.s3.us-east-2.amazonaws.com/bootstrap.zip) and unzip it into `volumes/factomd/main-database/ldb/MAIN/`
+
+
+#### 4. Config files
+The default file `config/factomd.conf` should work for most people, however feel free to edit it to fit your needs.
+
+
+#### 5. Start `factomd`
+
+Start `factomd` to finish syncing the blockchain
+
+```shell script
+docker-compose up -d factomd
+```
+
+#### 6. Check sync progress
+
+`factomd` will launch a cool control panel where you can check the progress of the sync: http://localhost:8090
+
+This might take a couple of hours if you used the bootstrap file, otherwise it might take over a day.
+
+You can also track it at the command line with the following command.
+
+```shell script
+docker-compose exec factomd curl -X POST --data-binary '{"jsonrpc": "2.0", "id": 0, "method": "heights"}' -H 'content-type:text/plain;' http://localhost:8088/v2
```

Comment:
We can improve on these curl commands by using `factom-cli`. Not a required change for this version but something we should think about in the future. 

https://hub.docker.com/r/factominc/factom-cli